test/elm/arithmetic/arithmetic-test.ts

```diff
@@ -159,36 +160,36 @@ describe('Divide', () => {
     setup(this, data);
   });
 
-  it('should divide two numbers', function () {
-    this.tenDividedByTwo.exec(this.ctx).should.equal(5);
+  it('should divide two numbers', async function () {
+    (await this.tenDividedByTwo.exec(this.ctx)).should.equal(5);
   });
 
-  it("should divide two numbers that don't evenly divide", function () {
-    this.tenDividedByFour.exec(this.ctx).should.equal(2.5);
+  it("should divide two numbers that don't evenly divide", async function () {
+    (await this.tenDividedByFour.exec(this.ctx)).should.equal(2.5);
   });
 
-  it('should divide multiple numbers', function () {
-    this.divideMultiple.exec(this.ctx).should.equal(5);
+  it('should divide multiple numbers', async function () {
+    (await this.divideMultiple.exec(this.ctx)).should.equal(5);
   });
 
-  it('should divide variables', function () {
-    this.divideVariables.exec(this.ctx).should.equal(25);
+  it('should divide variables', async function () {
+    (await this.divideVariables.exec(this.ctx)).should.equal(25);
   });
 
-  it('should divide uncertainty by uncertainty', function () {
-    const result = this.divideUncertainties.exec(this.ctx);
+  it('should divide uncertainty by uncertainty', async function () {
+    const result = await this.divideUncertainties.exec(this.ctx);
     result.low.should.equal(6 / 14);
     result.high.should.equal(9);
   });
 
-  it('should divide uncertainty by number', function () {
-    const result = this.divideUncertaintyByNumber.exec(this.ctx);
+  it('should divide uncertainty by number', async function () {
+    const result = await his.divideUncertaintyByNumber.exec(this.ctx);
```

Comment:
```suggestion
    const result = await this.divideUncertaintyByNumber.exec(this.ctx);
```
Please run these tests when you update them. This caused a descriptive TSC error that would have pointed this out. As simple as running `npm run test test/elm/arithmetic/arithmetic-test.ts`.